commands/health.md

```
@@ -0,0 +1,59 @@
+---
+description: Check Unraid system health and connectivity
+argument-hint: [action]
+---
+
+Execute the `unraid_health` MCP tool with action: `$1`
```

markdownlint-cli2 (0.20.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)